src/axom/lumberjack/TextEqualityCombiner.hpp

```diff
@@ -65,11 +65,7 @@ class TextEqualityCombiner : public Combiner
   bool shouldMessagesBeCombined(const Message& leftMessage,
                                 const Message& rightMessage)
   {
-    if(leftMessage.text().compare(rightMessage.text()) == 0)
-    {
-      return true;
-    }
-    return false;
+    return (leftMessage.text().compare(rightMessage.text()) == 0);
```

Comment:
This PR showed a bug in this combiner where depending on what is `leftMessage` and `rightMessage` the tag might be lost.

Reply:
Can you add a note that using this Combiner with tags has undetermined behavior?

Reply:
Warning added to `TextEqualityCombiner` about undefined behavior with tags.